migrations_azure/versions/d10c6bfdd9aa_put_lrf_table_back.py

```diff
@@ -0,0 +1,65 @@
+"""put lrf table back
+
+Revision ID: d10c6bfdd9aa
+Revises: 65c5753b57e0
+Create Date: 2021-01-11 10:31:39.441091
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mssql
+
+# revision identifiers, used by Alembic.
+revision = "d10c6bfdd9aa"
+down_revision = "65c5753b57e0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
This part here says to add code as needed here. Do we need to adjust any of this?